run_toshi_hazard_post_script.py

```diff
@@ -76,19 +77,19 @@
 ## Index 1: Only the highest weighted branch of the source logic tree and the full ground motion logic tree
 ## Index 2: The full source logic tree and only the highest weighted branch of the ground motion logic tree
 
-logic_tree_pair_list1 = [
-    full_logic_tree_pair,
-    logic_tree_tools.reduce_logic_tree_pair_to_nth_highest_branches(
-        full_logic_tree_pair,
-        source_logic_tree_nth_highest=1,
-        ground_motion_logic_tree_nth_highest=None,
-    ),
-    logic_tree_tools.reduce_logic_tree_pair_to_nth_highest_branches(
-        full_logic_tree_pair,
-        source_logic_tree_nth_highest=None,
-        ground_motion_logic_tree_nth_highest=1,
-    ),
-]
+# logic_tree_pair_list1 = [
+#     full_logic_tree_pair,
+#     logic_tree_tools.reduce_logic_tree_pair_to_nth_highest_branches(
+#         full_logic_tree_pair,
+#         source_logic_tree_nth_highest=1,
+#         ground_motion_logic_tree_nth_highest=None,
+#     ),
+#     logic_tree_tools.reduce_logic_tree_pair_to_nth_highest_branches(
+#         full_logic_tree_pair,
+#         source_logic_tree_nth_highest=None,
+#         ground_motion_logic_tree_nth_highest=1,
+#     ),
+# ]
```

Comment:
Delete?

Reply:
sorry, this was all commented out for debugging. Uncommented now.